en/config/safety_simulation.md

```diff
@@ -0,0 +1,27 @@
+# Failsafe State Machine Simulation
+
+This page can be used to simulate the actions of the PX4 failsafe state machine under all possible configurations and conditions.
+
+The simulation runs the same code in the browser as is executed on the vehicle, in real-time (the simulation is automatically kept in sync with the latest version of the code).
+Note that any delayed action (`COM_FAIL_ACT_T`) will also be delayed in the simulation.
+
+To use it:
+
+1. First configure the parameters on the left.
+   The initial values correspond to the PX4 defaults.
+2. Set the vehicle type
+3. Set the other values in the **State** or any of the flags under **Conditions**
+   - The **Intended Mode** corresponds to the commanded mode via RC or GCS (or external script).
+     The failsafe state machine can override this in case of a failsafe.
+4. Check the action under **Output**
+5. Check what happens when changing mode or **Move the RC sticks**
+6. Play with different settings and conditions!
+
+
+The simulation can also be executed locally:
+
+```sh
+make run_failsafe_web_server
```

Comment:
This didn't work for me on today's main and was not shown in `make list_config_targets`. 
- This is coming?
- I presume it runs a server and the command prompt shows you what URL to look at?
- This is useful if you want to try it with your own failsafe changes? i.e. why would you want to do this?

Reply:
The PR isn't merged yet, but you can test it there. It will tell you to first install the SDK then point you to the url.

> This is useful if you want to try it with your own failsafe changes? i.e. why would you want to do this?

Yes for development. Or if you use a specific version where you want to verify the exact behavior.

Reply:
OK. This will all work when you merge then. Sounds good.